src/Commands/core/ArchiveDumpCommands.php

```diff
@@ -0,0 +1,591 @@
+<?php
+
+namespace Drush\Commands\core;
+
+use Consolidation\SiteAlias\SiteAliasManagerAwareInterface;
+use Consolidation\SiteAlias\SiteAliasManagerAwareTrait;
+use Drush\Commands\DrushCommands;
+use Drush\Drush;
+use Drush\Sql\SqlBase;
+use Drush\Utils\FsUtils;
+use Exception;
+use FilesystemIterator;
+use Phar;
+use PharData;
+use RecursiveCallbackFilterIterator;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Yaml\Yaml;
+use Traversable;
+use Webmozart\PathUtil\Path;
+
+/**
+ * Class ArchiveDumpCommands.
+ */
+class ArchiveDumpCommands extends DrushCommands implements SiteAliasManagerAwareInterface
+{
+    use SiteAliasManagerAwareTrait;
```

Comment:
Use BootstrapManager instead of SiteAliasManager to get the root. Use a direct Drupal API to get the path to the files.

Reply:
Ok, will do

Reply:
Done